src/components/common/YourCompany/NameTranslations/NameTranslation.vue

```diff
@@ -6,15 +6,13 @@
     />
 
     <v-row no-gutters v-if="!isEditing">
-      <v-col cols="3">
+      <v-col cols="3" class="pr-2">
```

Comment:
This is provide some whitespace between this column and the next one, ie:

![image](https://user-images.githubusercontent.com/10002889/178563629-09fcc6cd-ea16-4a83-b56d-73a23bb3ddc1.png)

Same below.